test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,65 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class UtilsTest {
+
+    private Utils utils;
+    
+    @Before
+    public void setUp() throws Exception {
+        utils = new Utils();
+    }
+    
+    /*
+     * Tests for single object
+     */
+    
+    @Test
+    public final void test_isAnyNull_singleNullObject() {
+        Object object = null;
+        assertTrue(Utils.isAnyNull(object));
+    }
+
+    @Test
+    public final void test_isAnyNull_singleObject() {
+        Object object = new Object();
+        assertFalse(Utils.isAnyNull(object));
+    }
+    
+    @Test
+    public final void test_elementsAreUnique_singleObject() {
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.